css/pages/index.css

```diff
@@ -355,6 +375,7 @@ div.handles-container {
 .events {
   color: #604af0;
   margin: 0 0.25rem;
+  font-size: 1rem;
```

Comment:
Here also 1 rem is the default font size. We can remove this.

Reply:
made it 1.2rem